### DISCUSSION TITLE
Change acessory icon in bridge mode

### DISCUSSION THREAD

**Discussion by aLAN-LDZ** [HomeSpan/HomeSpan#225] [Q&A]
Can I change the accessories icon from switch to IrrigationSystem?

**Comment by HomeSpan** [comment]
The Home App controls the icons.  Did you set Category to Sprinklers in home.begin()?  Also, are you using just a Switch Service, or are you using the HAP IrrigationSystem Service?  If you are just using a plain switch service, the Home App only allows limited customization by clicking the icon (you can choose, light, fan, or outlet).

**Comment by aLAN-LDZ** [comment]
In home.begin() I have Bridge type because I have 5 accessories. Unfortunately, I use a regular switch because you can use it in automation as opposed to watering systems. I wanted to change the icon only because of a better look

**Comment by HomeSpan** [comment]
What you are trying to do makes perfect sense, and a big complaint about the Home App is its lack of customization.  You may want to check out the Eve for HomeKit App - it allows for some more customization of the icons.
